gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -318,14 +318,26 @@ boolean cancelIfStale() {
         switch (this.state) {
           case IDLE:
             if (!idleTimeout.isZero() && waitTime >= idleTimeout.toMillis()) {
-              myError = new WatchdogTimeoutException("Canceled due to idle connection", false);
+              myError =
+                  new WatchdogTimeoutException(
+                      "Canceled due to idle connection. waitTime="
+                          + waitTime
+                          + "ms idleTimeout="
+                          + idleTimeout.toMillis()
+                          + "ms",
```

Comment:
I would rephrase this:

"Canceled idle RPC after X ms (threshold: Y ms)

Reply:
Done.